interpreter/variable/all.go

```diff
@@ -777,8 +778,8 @@ func (v *AllScopeVariables) Get(s context.Scope, name string) (value.Value, erro
 		return &value.Integer{Value: 0}, nil
 	}
 
-	if val := v.getFromRegex(name); val != nil {
-		return val, nil
+	if val, err := v.getFromRegex(name); val != nil {
+		return val, err
```

Comment:
I think we need specific error handling because val may be `nil` even err is nil.

```suggestion
	if val, err := v.getFromRegex(name); err != nil {
		return nil, err
	} else if val != nil {
	    return val, nil
	}
```